tests/splash-response-google.json

```diff
@@ -8,7 +8,7 @@
+  "html": "<html></html>",
```

Comment:
Why this change?

Reply:
oh, that slipped in accidentally! need to revert that!

Reply:
done